mode_manager_cli.py

```
@@ -0,0 +1,56 @@
+"""
+Simple CLI access point for ModeManager feature demonstration and testing.
+"""
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)